src/main/java/sawkiewicz/task1/ItemController.java

```diff
@@ -0,0 +1,63 @@
+package sawkiewicz.task1;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+
+@RestController
+public class ItemController {
+    ItemRepository itemRepository = new ItemRepository();
+    ShoppingCartRepository shoppingCartRepository = new ShoppingCartRepository();
+
+
+
+    @RequestMapping(method = RequestMethod.GET, value = "/total/{cartId}")
+    ResponseEntity<String> showTotalPrice(@PathVariable String cartId) {
+        return ResponseEntity.ok("Total price is: " + shoppingCartRepository.findById(cartId).calculateTotalCost());
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/cart/{cartId}/item")
+    public ResponseEntity<String> addItem(@PathVariable String cartId, @RequestBody Item itemToAdd) {
+        CartItem cartItemToAdd = new CartItem(1, itemToAdd);
+
+        int sizeOfPurchase = shoppingCartRepository.findById(cartId).purchase.size();
+        System.out.println(sizeOfPurchase);
```

Comment:
don't use `println`, that's a bad practise

Reply:
removed